ekncontent/ekncontent/eknc-domain.c

```diff
@@ -633,6 +644,7 @@ eknc_domain_get_object (EkncDomain *self,
 
   GInputStream *stream = eos_shard_blob_get_stream (record->metadata);
```

Comment:
Worth using `g_autoptr(GInputStream) stream = …` and avoid `g_object_unref()` altogether.